mne/stats/regression.py

```diff
@@ -391,3 +394,9 @@ def _make_evokeds(coefs, conds, cond_length, tmin_s, tmax_s, info):
             kind='average')  # nave and kind are technically incorrect
         cumul += tmax_ - tmin_
     return evokeds
+
+
+def _solver(X, y):
+	a = (X.T * X).toarray()  # dot product of sparse matrices
```

Comment:
I think there's a fast_dot somewhere, but I don't know whether it's worth it here